src/crypto/qat/qcccrypto.h

```diff
@@ -28,6 +30,16 @@ extern "C" {
 #include "icp_sal_user.h"
 #include "icp_sal_poll.h"
 #include "qae_mem_utils.h"
+#else
+#include <qat/cpa.h>
```

Comment:
just realized that QATDrv installs the headers into a quite different path -- `quickassist/include` =(